docs/resources/escalation_path.md

```diff
@@ -129,6 +132,7 @@ resource "incident_escalation_path" "urgent_support" {
 
 ### Optional
 
+- `team_ids` (Set of String) IDs of teams that own this schedule
```

Comment:
Doesn't this mean that it will generate a different state every time?

Reply:
No, when it compares the current state to the remote state, it'll do so without caring about the order. That will prevent us incorrectly saying that there's a difference if the API returns the team IDs in a different order to what's in the state.